AirCasting/SessionViews/SessionCartView.swift

```diff
@@ -5,42 +5,41 @@
 //  Created by Lunar on 08/01/2021.
 //
 
-import SwiftUI
-import CoreLocation
-import CoreData
-import Charts
 import AirCastingStyling
+import Charts
+import CoreData
+import CoreLocation
+import SwiftUI
 
 struct SessionCartView: View {
-    
+    @State private var isFollowing = false
     @State private var isCollapsed = true
     @State private var selectedStream: MeasurementStreamEntity?
-    
+    @EnvironmentObject var persistenceController: PersistenceController
```

Comment:
`PersistenceController` should not be present in here. The whole purpose of `MeasurementStreamStorage` protocol is so we don't use `CoreData` outside of its implementation. If you really need to use storage here it should be injected from outside and used as a `MeasurementStreamStorage`